cisco_sdwan/assets/monitors/tunnel_down.json

```diff
@@ -0,0 +1,31 @@
+{
+	"version": 2,
+	"created_at": "2024-04-05",
+	"last_updated_at": "2024-04-05",
+	"title": "Cisco SD-WAN Tunnel is down",
+	"tags": [
+		"integration:cisco-sdwan"
+	],
+	"description": "This monitor detects when a Cisco SD-WAN tunnel becomes unavailable between two devices. It tracks the status of tunnels within a specified namespace and triggers an alert if a tunnel goes down, indicating potential connectivity issues.",
+	"definition": {
+		"name": "[Cisco SD-WAN] Tunnel is down between {{local_color.name}} {{hostname.name}} ({{system_ip.name}}) to {{remote_color.name}} {{remote_hostname.name}} ({{remote_system_ip.name}})",
+		"type": "query alert",
+		"query": "min(last_15m):min:cisco_sdwan.tunnel.status{*} by {device_namespace,system_ip,hostname,local_color,remote_system_ip,remote_hostname,remote_color} < 1",
```

Comment:
Similarly, just to confirm `cisco_sdwan.tunnel.status` is fully defined here so there is no space aggregation. Correct?

Reply:
This might be a bit too sensitive as it will trigger if the tunnel is down just once.
How many data points can we expect from this metric, is it going to be very sparse?
We may want to use an `avg` time-rollup and trigger on some value `< 1` that we define.
This has the benefit of being more easily customizable by a user, where they can increase/decrease the threshold as needed

Reply:
The metric is fully defined by `(system_ip, local_color, remote_color, remote_system_ip)`
This is a very sparse metric (10 mins interval) so time aggregation on a 15mins interval shouldn't impact evaluation
Given the high interval I think it's worth alerting any time we see a `0`

Reply:
👍